### DISCUSSION TITLE
Omit some relationships without "within" keyword

### DISCUSSION THREAD

**Discussion by NaumenkoSergiy** [okuramasafumi/alba#337] [General]
I have to serializers which cause "SystemStackError (stack level too deep)" error:

```
class DisclosureSerializer < BaseSerializer
  ..................
  many :agreements
end

class AgreementSerializer < BaseSerializer
  ...................
  one :disclosure
end
```

I can write something like:

```
class AgreementSerializer < BaseSerializer
  ...................
  one :disclosure do
    attributes ...........
  end
end
```

but is it possible to write something like `exclude`? Because I have a lot of attributes and relationships, and just want to ignore one relationships. Or maybe there is general solution to avoid "SystemStackError (stack level too deep)" error?

**Comment by okuramasafumi** [comment]
@NaumenkoSergiy Why do you want to avoid `within`? It's the way to deal with circular associations like this.
